server/webapp/WEB-INF/vm/auth/login.vm

```diff
@@ -123,6 +123,11 @@
                 </div>
                 #end
                 <div class='clear-both'><!-- Clear floats --></div>
+
+#foreach ($authentication_plugin_id in $authentication_plugin_registry.getAuthenticationPlugins())
```

Comment:
I thought about it. If we list all available authentication schemes on the default login page then the user can choose to use what he prefers. admins can prefer authentication schemes by configuring LDAP/Password/adding the right plugin. I suppose that much flexibility should be enough?